lib/improver/tests/utilities/cube_manipulation/test_merge_cubes.py

```diff
@@ -33,23 +33,16 @@
 """
 
 import unittest
+import numpy as np
 from datetime import datetime as dt
 
 import iris
 from iris.cube import Cube
 from iris.exceptions import DuplicateDataError, MergeError
 from iris.tests import IrisTest
-import numpy as np
```

Comment:
I'm happy there are no extra changes in this file apart from those caused by moving this test file from cube_manipulation_new to cube_manipulation